# Change the Makefile to make it work with Gentoo and to make it install the shell completions.
# The "${D}" parameter is used at the start of the installation paths to respect the Portage sandbox.

--- shell-color-scripts-source/Makefile
+++ shell-color-scripts-patched/Makefile
@@ -1,12 +1,6 @@
-clean: 
-	rm -rf /opt/shell-color-scripts
-
-install: clean
-	mkdir -p /opt/shell-color-scripts/colorscripts
-	cp -rf colorscripts/* /opt/shell-color-scripts/colorscripts
-	cp colorscript.sh /usr/local/bin/colorscript
-
-uninstall:
-	rm -rf /opt/shell-color-scripts
-	rm -f /usr/local/bin/colorscript
+install:
+	install -Dm755 completions/_colorscript ${D}/usr/share/zsh/_colorscript
+	install -Dm755 completions/colorscript.fish ${D}/usr/share/fish/vendor_completions.d/colorscript.fish
+	install -Dm755 colorscripts/* -t ${D}/opt/shell-color-scripts/colorscripts
+	install -Dm755 colorscript.sh ${D}/usr/local/bin/colorscript 

